examples/analytics/emr-on-eks/main.tf

```
@@ -92,7 +92,10 @@ module "eks_blueprints" {
 module "eks_blueprints_kubernetes_addons" {
   source = "../../../modules/kubernetes-addons"
 
-  eks_cluster_id = module.eks_blueprints.eks_cluster_id
+  eks_cluster_id           = module.eks_blueprints.eks_cluster_id
+  aws_eks_cluster_endpoint = module.eks_blueprints.eks_cluster_endpoint
+  eks_oidc_issuer_url      = module.eks_blueprints.eks_oidc_issuer_url
```

see note below on changing `eks_oidc_issuer_url` to `eks_oidc_provider` and then this becomes
```suggestion
  eks_oidc_provider     = module.eks_blueprints.eks_oidc_provider
```

Since we don't want to use the URL attribute that is preceded with `https://` in the assume role with web identity of IRSA assume role policies